common/actions/swap.js

```diff
@@ -1,86 +1,103 @@
+// @flow
 import {
   SWAP_DESTINATION_AMOUNT,
   SWAP_DESTINATION_KIND,
   SWAP_ORIGIN_AMOUNT,
   SWAP_ORIGIN_KIND,
   SWAP_UPDATE_BITY_RATES,
-  SWAP_PART_ONE_COMPLETE,
-  SWAP_PART_TWO_COMPLETE,
   SWAP_DESTINATION_ADDRESS,
   SWAP_RESTART,
   SWAP_LOAD_BITY_RATES,
-  SWAP_STOP_LOAD_BITY_RATES
+  SWAP_STOP_LOAD_BITY_RATES,
+  SWAP_STEP,
+  SWAP_REFERENCE_NUMBER
 } from './swapConstants';
```

Comment:
You only need to autocomplete twice: when writing action creator and when writing reducer. Not sure if worth